plugins/gfonts.koplugin/featured.lua

```diff
@@ -0,0 +1,61 @@
+local Featured = {
+    -- recommended fonts, both from gfonts and custom repos
+    fonts = {
+	"Bitter",
+        "Crimson Text",
+        "Gentium Book Basic",
+        "Ibarra Real Nova",
+        "Literata",
+        "Merriweather",
+        "Source Serif Pro",
+    },
+
+    -- we'll use custom versions of the following font families:
+    own_fonts = {
+	-- NiLuJe build of Literata 3 goes here
```

Comment:
I don't think the license allows redistribution of modified files?

Reply:
Well, not exactly. It's OFL, so, it can, but not under the name "Literata". Incidentally, I'm currently renaming it to "Lit3rata" (originally purely to be able to compare it to Literata 2 :D). Dunno if that's too cheap to count ;p.

Reply:
I see no problem with SIL OFL. See https://theleagueof.github.io/licenses/ofl-faq.html (specially chapter 3)

Reply:
In which case:

https://storage.sbg.cloud.ovh.net/v1/AUTH_2ac4bfee353948ec8ea7fd1710574097/koreader-pub/Lit3rata-Regular.otf
https://storage.sbg.cloud.ovh.net/v1/AUTH_2ac4bfee353948ec8ea7fd1710574097/koreader-pub/Lit3rata-Italic.otf
https://storage.sbg.cloud.ovh.net/v1/AUTH_2ac4bfee353948ec8ea7fd1710574097/koreader-pub/Lit3rata-Bold.otf
https://storage.sbg.cloud.ovh.net/v1/AUTH_2ac4bfee353948ec8ea7fd1710574097/koreader-pub/Lit3rata-BoldItalic.otf

(I'd add a link to http://trac.ak-team.com/trac/browser/niluje/Configs/trunk/Kindle/Kobo_Hacks/Patches/REMINDER#L125 in the comments).

And, FWIW, I'm using Regular for Regular, not Medium ;).

Reply:
The metadata looks sane for a Modified Version, if I compare it to the Bitter/BitterPro situation ;).

Reply:
Included in last commit. It does look nice!!!